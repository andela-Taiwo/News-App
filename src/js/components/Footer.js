import React from 'react';


export default class Footer extends React.Component {
    render() {
        return (
            <div className="page-footer ">
                <div className="container">

              <div className="footer-copyright center"> 
                  <div className="container">
                  Sokunbi Memunat Â©2017.
                  </div>
                  </div>
                    
                </div>

            </div>
        );
    }
}
