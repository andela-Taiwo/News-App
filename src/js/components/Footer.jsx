import React from 'react';

/**
 * @export
 * @returns {any}
 */
export default function Footer() {
  return (
    <div className="footer">
        Sokunbi Memunat Â©2017.
    </div>
  );
}
