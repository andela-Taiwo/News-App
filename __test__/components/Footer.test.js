import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../src/js/components/Footer.jsx';


describe('Footer', () => {
  it('Found footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('div').length).toEqual(1);
  });
});

describe('Footer', () => {
  it('Found div', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('div').text()).toBe('Sokunbi Memunat ©2017.');
  });
});

describe('Footer (Snapshot)', () => {
  it('Footer renders Sokunbi Memunat ©2017', () => {
    const component = renderer.create(<Footer />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});

