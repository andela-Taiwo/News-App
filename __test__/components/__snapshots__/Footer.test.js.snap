// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer (Snapshot) Footer renders Sokunbi Memunat ©2017 1`] = `
<div
  className="footer"
>
  Sokunbi Memunat ©2017.
</div>
`;
